management/user.go

```diff
@@ -343,3 +383,28 @@ func (m *UserManager) Blocks(id string, opts ...RequestOption) ([]*UserBlock, er
 func (m *UserManager) Unblock(id string, opts ...RequestOption) error {
 	return m.Request("DELETE", m.URI("user-blocks", id), nil, opts...)
 }
+
+// RegenerateRecoveryCode removes the current multi-factor authentication recovery code and generates a new one.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/post_recovery_code_regeneration
+func (m *UserManager) RegenerateRecoveryCode(id string) (*UserRecoveryCode, error) {
+	r := new(UserRecoveryCode)
+	err := m.Request("POST", m.URI("users", id, "recovery-code-regeneration"), r)
+	return r, err
+}
+
+// Enrollments retrieves all Guardian enrollments for a user.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/get_enrollments
+func (m *UserManager) Enrollments(id string) (enrols []*UserEnrollment, err error) {
+	err = m.Request("POST", m.URI("users", id, "enrollments"), &enrols)
+	return enrols, err
+}
+
+// ListAuthenticators retrieves all authenticators for a user.
+//
+// It's an undocumented API, see: https://community.auth0.com/t/when-will-api-v2-users-userid-authenticators-be-documented/52722
+func (m *UserManager) ListAuthenticators(id string) (enrols []*UserAuthenticator, err error) {
+	err = m.Request("POST", m.URI("users", id, "authenticators"), &enrols)
```

Comment:
Hello and thanks for making these additions! Just wanted to pop by and make a quick comment. Using POST for the request in `ListAuthenticators` gave me a 404, but using GET works. I think we might need to use GET instead (it's probably the same for `Enrollments` as well).

Reply:
Yikes, thanks, That's why committing stuff late at night is a terrible idea.